README.md

```diff
@@ -238,7 +241,84 @@ govuk-docker prune
 
 You should then be able to `govuk-docker build` your service and have confidence you're not suffering from configuration drift.
 
-### How to: set up Dnsmasq manually
+### How to: resolve setup issues
+
+The following sections resolve common installation issues.
+
+#### How to: update Ruby
+
+Update Ruby using [brew](https://brew.sh/), then add it to the PATH in your bash config.
+
+```
+brew install ruby
+echo 'export PATH="/usr/local/opt/ruby/bin:$PATH"' >> ~/.bash_profile
+```
+
+Restart your terminal, or use `source ~/.bash_profile` to activate the new Ruby version; you can check that you have the new version using `ruby -v`.
+
+#### How to: install bundler
+
+To install [bundler](https://bundler.io/), first find out the required version (in X.Y.Z format) from [Gemfile.lock](https://github.com/alphagov/govuk-docker/blob/master/Gemfile.lock) - it will be listed as:
+
+```
+BUNDLED WITH
+    X.Y.Z
+```
+
+Install the correct version of bundler; you may have to overwrite existing bundle and bundler executable conflicts.
+
+```
+gem install bundler:X.Y.Z
+```
+
+#### How to: resolve issues caused by an existing docker install 
+
+During `govuk-docker setup`, if get the follow errors when pouring `docker-compose`:
+
+```
+Error: The `brew link` step did not complete successfully
+The formula built, but is not symlinked into /usr/local
+Could not symlink bin/docker-compose
+Target /usr/local/bin/docker-compose
+already exists. You may want to remove it:
+  rm '/usr/local/bin/docker-compose'
+
+To force the link and overwrite all conflicting files:
+  brew link --overwrite docker-compose
+
+To list all files that would be deleted:
+  brew link --overwrite --dry-run docker-compose
+
+Possible conflicting files are:
+/usr/local/bin/docker-compose -> /Applications/Docker.app/Contents/Resources/bin/docker-compose
+```
+
+and when pouring `docker`:
+
+```
+Error: It seems there is already an App at '/Applications/Docker.app'.
+```
+
+Then uninstall your existing docker, and restart the `govuk-docker setup` process to install a new version of docker using brew.
+
+#### How to: resolve `No such file or directory` errors for `dev.gov.uk`
+
+If you get the following error during `govuk-docker setup`:
```

Comment:
There's a PR to fix this error fully at #160, but these docs are good for now. 👍